kuma/users/urls_beta.py

```diff
@@ -0,0 +1,35 @@
+import importlib
+
+from allauth.account import views as account_views
+from allauth.socialaccount import providers
+from django.conf.urls import include, url
+from django.views.decorators.csrf import csrf_exempt
+
+from kuma.core.decorators import redirect_in_maintenance_mode
+
+
+urlpatterns = []
+for provider in providers.registry.get_list():
+    try:
+        prov_mod = importlib.import_module(provider.package + '.urls')
+    except ImportError:
+        continue
+    prov_urlpatterns = getattr(prov_mod, 'urlpatterns', None)
+    if prov_urlpatterns:
+        urlpatterns += prov_urlpatterns
+
+users_patterns = [
+    url(r'^signup/?$',
```

Comment:
I don't think this one is used any longer? I'll look into it some more, but just wanted to document my thought for now. https://developer.mozilla.org/en-US/users/signup returns a page saying it has been disabled, but I don't know how one even gets to that page.

Reply:
Good point. I've removed it @escattone.